vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -1503,4 +1496,19 @@ - (void)onContributeUnverifiedPublishers:(ledger::Result)result publisherKey:(co
   }
 }
 
+- (void)getExternalWallets:(ledger::GetExternalWalletsCallback)callback
+{
+
+}
+
+- (void)saveExternalWallet:(const std::string &)wallet_type wallet:(ledger::ExternalWalletPtr)wallet
+{
+
+}
+
+- (void)showNotification:(const std::string &)type args:(const std::vector<std::string>&)args callback:(const ledger::ShowNotificationCallback&)callback
+{
+
```

Comment:
Are there any implications of not calling the callback? If so, perhaps we should call it with a failure for now?

Reply:
it's not needed